src/raw_field.rs

```diff
@@ -39,6 +39,17 @@ macro_rules! _memoffset__addr_of {
 }
 
 /// Deref-coercion protection macro.
+///
+/// Prevents complilation if the specified field name is not a part of the
+/// struct definition.
+///
+/// ```compile_fail
+/// use memoffset::_memoffset__field_check;
+/// struct Foo {
+///     foo: i32,
+/// }
+/// _memoffset__field_check!(Box<Foo>, foo);
```

Comment:
Can you add a type alias for `Box<Foo>`? Just to ensure it's not the `<...>` that makes it fail...

Reply:
Great idea!